src/tests/system/tests/test_files.py

```diff
@@ -13,65 +16,75 @@
 from sssd_test_framework.topology import KnownTopology
 
 
+@pytest.mark.importance("low")
 @pytest.mark.builtwith("files-provider")
 @pytest.mark.topology(KnownTopology.Client)
-def test_files__lookup_root(client: Client):
+def test_files__get_entry_using_sss_service_should_not_work_looking_for_root(client: Client):
     """
-    :title: Getent call doesnt work on root, when service specified as "sss"
+    :title: Get entry using sss service should not work looking for root
     :setup:
-        1. Enable files domain
+        1. Configure SSSD with files provider
         2. Start SSSD
     :steps:
-        1. getent passwd -s sss root
+        1. Lookup root user using sss service
+        2. Lookup root user without the sss service
     :expectedresults:
-        1. Call failed
+        1. The root user is not found
+        2. The root user is found
     :customerscenario: False
     """
     client.sssd.sssd["enable_files_domain"] = "true"
     client.sssd.start()
 
     result = client.tools.getent.passwd("root", service="sss")
-    assert result is None, "Getent call was successful, which is not expected"
```

Comment:
Please do not remove the error messages, rather make sure that they are consistent with the
expectedresults